src/components/base/outline-button/test/outline-button.test.ts

```diff
@@ -1,7 +1,12 @@
 import { OutlineButton } from '../outline-button';
-import { assert, fixture, html } from '@open-wc/testing';
+import { assert, expect, fixture, html } from '@open-wc/testing';
+import { ButtonVariant, ButtonSize, ButtonType } from '../outline-button';
```

Comment:
Couldn't we import these as part of line 1?

Reply:
yep...